magento/adobe-stock-integration
# Performed media gallery && media content synchronization in batches

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1337:
Media gallery synchronization should be performed in batches
2.  magento/adobe-stock-integration#1338: Media content synchronization should be performed in batches


### Manual testing scenarios (*)
1. Run `bin/magento media-gallery:sync`
2. `php bin/magento media-content:sync`

Media gallery synchronization executed with batch of files 
Verify all files synchronized correctly
